README.md

```diff
@@ -22,6 +22,9 @@
   <a href="https://twitter.com/intent/follow?screen_name=reactnative">
     <img src="https://img.shields.io/twitter/follow/reactnative.svg?label=Follow%20@reactnative" alt="Follow @reactnative" />
   </a>
+  <a href="https://bsky.app/profile/reactnative.dev">
+    <img src="https://img.shields.io/badge/Bluesky-0285FF?logo=bluesky&logoColor=fff" alt="Follow @reactnative.dev" />
```

Comment:
would it be worth tweaking the alt text of the two badges to specify the sites? (e.g. "Follow @reactnative.dev on Bluesky")

this seems less confusing to me if you only hear the image's alt text, though if screen readers also read out the actual link then this probably isn't necessary but yeah

Reply:
Good point, thanks for the suggestion! 👍